src/pkg/oci/pull.go

```diff
@@ -216,20 +224,31 @@ func (o *OrasRemote) PullLayer(desc ocispec.Descriptor, destinationDir string) e
 	return utils.WriteFile(filepath.Join(destinationDir, desc.Annotations[ocispec.AnnotationTitle]), b)
 }
 
-// PullPackageMetadata pulls the package metadata from the remote repository and saves it to `destinationDir`.
-func (o *OrasRemote) PullPackageMetadata(destinationDir string) (err error) {
+// PullMultipleFiles pulls multiple files from the remote repository and saves them to `destinationDir`.
+func (o *OrasRemote) PullMultipleFiles(paths []string, destinationDir string) ([]ocispec.Descriptor, error) {
```

Comment:
```suggestion
func (o *OrasRemote) PullPackagePaths(paths []string, destinationDir string) ([]ocispec.Descriptor, error) {
```
PullPackagePaths might be a better way of naming this